scripts/remove-redirects.js

```diff
@@ -0,0 +1,78 @@
+const glob = require('glob');
+const {
+  writeMdx,
+  replaceFrontmatter,
+  removeRedirect,
+  getSlugFromPath,
+  yamlToJson,
+  getFrontmatter,
+  mdxToVfile,
+  jsonToYaml,
+  getYamlFromFrontmatter,
+} = require('./utils/edit-frontmatter');
+const toMDX = require('./utils/migrate/to-mdx');
+
+const processFiles = async (filepath) => {
+  const slug = getSlugFromPath(filepath);
+  const fileTree = await mdxToVfile(filepath);
+  const frontmatter = getFrontmatter(fileTree);
+  const yamlContent = getYamlFromFrontmatter(frontmatter);
+  const yamlJson = yamlToJson(yamlContent);
+  const frontmatterJson = yamlJson[0];
+  const { redirects } = frontmatterJson;
+
+  if (!redirects || !redirects.includes(slug)) {
+    return { skipped: slug };
+  }
```

Comment:
A lot of the `processFile` function (and the `main` function) is pretty similar in both the `add-redirects.js` and `remove-redirects.js` files. We might be able to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up this file a little more by moving this logic to the `utils/edit-frontmatter.js` file.

Perhaps something like this?

```js
// function for processing one directory / filepath
const processDir = (processFn) => async (filepath) => {
  // do all the common stuff
  const editedFrontMatter = processFn(frontmatterJson, slug);
  // do the save stuff
  return { processed: slug };
}

// function for processing all the files (e.g. your "main" function)
const processFiles = async (processFn) => {
  // glob stuff
  const processPromises = filepaths.map(processDir(processFn)); // abbreviated, but you could write this out
  const processResults = await Promise.all(processPromises);
  // added / removed files
  const modifiedFiles = processResults.filter(result => result.processed); // cloud be simplified further with a "prop" helper function
  // print stuff
}
```